File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/redis/FieldVo.java
package org.javamaster.invocationlab.admin.model.redis;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 */
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class FieldVo {
    private Integer fieldIndex;
    private String fieldKey;
    private Class<?> fieldKeyClazz;
    private Boolean fieldKeyJdkSerialize;

    private String fieldValue;
    private Boolean fieldValueJdkSerialize;
    private Class<?> fieldValueClazz;
    private Double fieldScore;
    private Integer fieldValueSize;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/redis/FieldVo.kt
package org.javamaster.invocationlab.admin.model.redis

import org.javamaster.invocationlab.admin.annos.AllOpen


/**
 * @author yudong
 */
@AllOpen
class FieldVo {
    var fieldIndex: Int? = null
    var fieldKey: String? = null
    var fieldKeyClazz: Class<*>? = null
    var fieldKeyJdkSerialize: Boolean? = null

    var fieldValue: String? = null
    var fieldValueJdkSerialize: Boolean? = null
    var fieldValueClazz: Class<*>? = null
    var fieldScore: Double? = null
    var fieldValueSize: Int? = null
}
